index.html

```diff
@@ -287,14 +597,65 @@ <h2>Grammar Extensions</h2>
             resolves to a single property/value relationship.</li>
         </ul>
       </li>
+      <li>A <a>node object</a> or a <a>value object</a> MAY contain an `@annotation` property (or an alias) referencing
+        zero or more <a>annotation objects</a>.</li>
+      <li>A <a>node object</a> which is not the value of a non-keyword property,
+        or which is an value of a <a>list object</a>,
+        MUST NOT contain an `@annotation` property.</li>
+      <li>An <a>annotation object</a> is a <a>node object</a> with the restriction
+        that it MUST NOT contain an `@id` <a>entry</a>.</li>
     </ul>
   </section>
 
   <section>
     <h2>Algorithmic Extensions</h2>
 
+    <p>This specification extends the altorithms from [[[JSON-LD11-API]]] [[JSON-LD11-API]]
+      to support the addition of <a>embedded nodes</a> and <a>annotation objects</a>.</p>
+
+    <p>The following algorithm extensions require that the <em>rdfstar</em> flag be set.</p>
+
     <section>
       <h3>Expansion Algorithm</h3>
+
+      <p>Before step 13.4.3.1, and changing 13.4.3.1 to be an "Otherwise, if",
+        add the following steps:</p>
+      <ol>
+        <li>If the <em>rdfstar</em> flag is not set,
+          and <var>expanded active property</var> is `@annotation`,
+          an <em>invalid annotation</em> error has been detected and processing is aborted.</li>
+        <li>If <var>value</var> is a non-empty <a>map</a> and
+          the <em>rdfstar</em> flag is set,
+          set <var>expanded value</var> to the result of using this algorithm recursively
+          passing <var>active context</var>, <var>active property</var>,
```

Comment:
```suggestion
          passing <var>active context</var>, `null` for <var>active property</var>,
```
otherwise you might have undesired side effects (imagine that the active property is `@annotation` and the embedded node has an `@id`...).